boards/arm/96b_ivy5661/board.h

```diff
@@ -0,0 +1,12 @@
+/*
```

Comment:
Remove board.h, shouldn't be needed anymore.

Reply:
fixed.